### ISSUE TITLE
Record training settings

### ISSUE THREAD

**Issue by jacaseyclyde** [sjsu-high-z-classify/GalNet#30]
**Is your feature request related to a problem? Please describe.**
Right now the training settings for a given model need to be recorded manually (i.e. in a comment documenting the results of any training run), making it easier for these values to be accidentally lost.

**Describe the solution you'd like**
We should record parameter settings (such as number of epochs, initial learning rate, optimizer used) in a file in the same folder as the model.

**Additional context**
While the settings suggested above would be a good place to start, there are likely many more settings which could be recorded as well.
